tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.md

```diff
@@ -0,0 +1,25 @@
+Test 5-30 - Docker Compose ELK
+=======
+
+# Purpose:
+To verify that VIC appliance can work when deploying the docker ELK services
+
+# References:
+[1 - Docker Compose Overview](https://docs.docker.com/compose/overview/)  
+[2 - Docker compose ELK](https://blogs.vmware.com/cloudnative/2018/07/19/getting-started-with-elastic-stack-on-vsphere-integrated-containers/)
+
+# Environment:
+This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
+
+# Test Steps:
+1. Deploy a new vCenter with 2 ESXi hosts in a cluster
+2. Deploy VIC appliance to the vSphere server 
+3. Issue the following command in the docker voting app folder:  
```

Comment:
Please rename "docker voting app folder" to the elk folder

Reply:
Currently not needed.